content/en/docs/reference/setup-tools/kubeadm/kubeadm-config.md

```diff
@@ -16,6 +16,11 @@ You can execute `kubeadm config view` to view the ConfigMap. If you initialized
 kubeadm v1.7.x or lower, you must use `kubeadm config upload` to create the ConfigMap before you
 may use `kubeadm upgrade`.
 
+In Kubernetes v1.11.0, some new commands were added. You can use `kubeadm config print-default`
+to print the default configuration and `kubeadm config migrate` to convert your old configuration
+files to a newer version. `kubeadm config images list` and `kubeadm config images pull` can be used
+to to list and pull the images which kubeadm requires.
+
```

Comment:
Sorry, I should have caught this on first review. Remove extra "to" on line 22.

But also -- do we need most of this content (ll19-22)? Hugo automatically generates a TOC list of all the commands based on their headings (see the Netlify preview). Does it help the user to know which ones are new at the top of the file? If so, do they need a summary of what they do when the names indicate pretty clearly what they do? I am not really sure how to make this work nicely with Hugo, will dig later.

Reply:
@Bradamant3 

thanks for spotting that typo,
so the user knowing that new commands are added (i.e. like in a "what's new" section) is definitely a great plus. i really do not know how this can be done automatically Hugo, but please let me know if you want me to remove the above notes.

i will wait on your decision before submitting the typo fix.

Reply:
Let's just fix the typo, and ponder alternatives to the list of new things another time.